src/components/overview/overview.js

```diff
@@ -6,15 +6,15 @@ import Cart from './cart.js'
 import Grid from '@mui/material/Grid'
 import Container from '@mui/material/Container'
 
-export default function Overview() {
+export default function Overview(props) {
```

Comment:
destructure on the spot if you don't use anything other than a single `props` property. 

```react.js
export default function Overview({ onClick }) { ... }
```